project/tsgd_json.ts

```diff
@@ -95,6 +98,7 @@ export class Paths {
       "dynamic"
     )
 
+    this.removeComments = tsgdJson.removeComments || false
```

Comment:
nit: ?? (maybe change the other line too :P)

Reply:
You mean why this is in `Paths` class or why this is called like that?

It is in `Paths` because I couldn't find another class which have access to `ts2gd.json` file and I wanted comment generation to be configurable from config.

Reply:
Oh I just mean instead of || we can use ?? e.g. `this.removeComments = tsgdJson.removeComments ?? false`

Reply:
Right, still getting used to newer TypeScript after working in old JS code :P

Reply:
:D I know that feeling. Welcome to the future my friend!!